bpf/kmesh/workload/include/authz.h

```diff
@@ -22,6 +23,25 @@ struct {
     __uint(max_entries, MAP_SIZE_OF_AUTH_POLICY);
 } map_of_authz SEC(".maps");
 
+struct match_result {
+    __u32 action;
+    __u32 match_res;
+    __u16 dport;
```

Comment:
if dport is needed？ this can be get from element "match"

Reply:
> if dport is needed？ this can be get from element "match"

We need to know whether it is ipv4 or ipv6 type. If we do not pass dport, we need to pass xdp_info to determine whether it is v4 or v6.